src/core/Text/Text.md

```diff
@@ -3,7 +3,7 @@ import { Paragraph, Text } from 'suomifi-ui-components';
 
 <>
   <Paragraph>Paragraph text</Paragraph>
-  <Paragraph marginBottomSpacing="insetXl">
+  <Paragraph marginBottomSpacing="xl">
```

Comment:
This is quite a bit larger than insetXl. Works like this as well, but if you want this to stay about the same, then use s or m as the value.

Reply:
Yes, I can change it to be `s` or `m`. Thought that it has no real purpose as it is just as an example which could use every possible token available.

Reply:
This is now changed to use the `s` token. It was the closest size of the original author's wanted size.